tests/model/test_model.py

```diff
@@ -1516,12 +1516,20 @@ def test_is_muted_stream(self, muted_streams, stream_id, is_muted,
         ((1, 'muted stream muted topic'), True),
         ((2, 'unmuted topic'), False),
     ])
-    def test_is_in_muted_topics(self, topic, is_muted, stream_dict, model):
+    def test_is_in_muted_topics(self, topic, is_muted, stream_dict, model,
+                                zulip_version):
         model.stream_dict = stream_dict
-        model.muted_topics = [
-            ['Stream 2', 'muted topic'],
-            ['Stream 1', 'muted stream muted topic'],
-        ]
+        server_version, server_feature_level = zulip_version
+        if server_version == '3.0' and server_feature_level >= 1:
+            model.muted_topics = [
+                ['Stream 2', 'muted topic', 1530129122],
+                ['Stream 1', 'muted stream muted topic', 1530129122],
+            ]
+        elif server_version == '2.1':
+            model.muted_topics = [
+                ['Stream 2', 'muted topic'],
+                ['Stream 1', 'muted stream muted topic'],
+            ]
```

Comment:
I'd prefer that we had this version-dependent data in a fixture, if possible. Also can we depend upon just the feature level?